packages/synthetic-chain/public/upgrade-test-scripts/run_eval.sh

```diff
@@ -13,22 +13,24 @@ fi
 
 startAgd
 
-echo "[$PROPOSAL] Agd started. Running CoreEval submission."
+echo "[$PROPOSAL] Agd started. Submitting proposal."
 cd /usr/src/proposals/"$PROPOSAL/" || fail "Proposal $PROPOSAL does not exist"
 
 if [ -f "eval.sh" ]; then
-    # this is what the script used to do. Also allows a proposal to override how they are eval-ed
+    # Allows a proposal to control its evaluation; used in particular for
+    # param-change proposals but also valid for other types such as core-eval.
     echo "[$PROPOSAL] Running eval.sh"
     ./eval.sh
 else
-    # newer proposals declare a submission
-    echo "[$PROPOSAL] Running proposal declared in package.json"
+    # Straightforward core-eval proposals can be specified directly in a "submission" directory
+    # containing $name.js scripts, $name-permit.json permits, and b1-$hash.json bundles.
+    echo "[$PROPOSAL] Running core-evals from submission/"
     # copy to run in the proposal package so the dependencies can be resolved
     cp /usr/src/upgrade-test-scripts/eval_submission.js .
     ./eval_submission.js
 fi
 
-echo "[$PROPOSAL] Eval completed. Running 10 blocks and exiting."
+echo "[$PROPOSAL] Proposal submitted. Running 10 blocks and exiting."
```

Comment:
It's not just submitted, it has begun execution, right?

A _submitted_ proposal may not be accepted. In this case it it is and the eval within the proposal started running.

Reply:
Good point. Updated the message.